frontend/src/Metamaps/Views/TopicCard.js

```diff
@@ -0,0 +1,61 @@
+/* global $ */
+
+import React from 'react'
+import ReactDOM from 'react-dom'
+
+import Active from '../Active'
+import DataModel from '../DataModel'
+import GlobalUI from '../GlobalUI'
+import Mapper from '../Mapper'
+import Router from '../Router'
+import Visualize from '../Visualize'
+
+import ReactTopicCard from '../../components/TopicCard'
+
+const TopicCard = {
+  openTopicCard: null, // stores the topic that's currently open
+  init: function() {
+    // initialize topic card draggability and resizability
+    $('.showcard').draggable({
+      handle: '.metacodeImage',
+      stop: function() {
+        $(this).height('auto')
+      }
+    })
+  },
```

Comment:
will this remain draggable after a re-render? what's the html?

```
<div class="showcard">
  <div id="showcard">
  </div>
</div>
```

is that right? and react renders into the inner one?

what's draggable? the whole card?